roles/nova-common/templates/etc/nova/policy.json

```diff
@@ -111,9 +111,9 @@
     "compute_extension:admin_actions:resetNetwork": "rule:admin_api",
     "compute_extension:admin_actions:injectNetworkInfo": "rule:admin_or_projAdmin_or_cloudAdmin",
     "compute_extension:admin_actions:createBackup": "rule:admin_or_projAdmin_or_cloudAdmin_or_owner",
-    "compute_extension:admin_actions:migrateLive": "rule:admin_or_projAdmin_or_cloudAdmin",
+    "compute_extension:admin_actions:migrateLive": "rule:admin_api",
```

Comment:
Live migration is a thing we are exposing to customers. It's not reliable, it doesn't honor scheduling constraints, and it's not a thing we want our users to depend on. We've only implemented it to facilitate host evacuations in the case of disruptive maintenance, and those are only to be initiated by IBM admins.